### DISCUSSION TITLE
Using PactumJS behind Corporate Proxy

### DISCUSSION THREAD

**Discussion by meshuggahbobo** [pactumjs/pactum#240] [Q&A]
Hi !

It's possibile to use or setup my testing project and execute all the request that i made through a corporate proxy?

Thank you so much

**Comment by divyaanand20** [comment]
Hi,
Had the same question, if its possible to use a proxy with Pactum JS? If yes, how can we do it?

Thank you

**Reply by leelaprasadv** [reply]
Hi @divyaanand20 - can you elaborate on your use case? 

- In which use case do you see a need for proxy?

**Reply by divyaanand20** [reply]
hi @leelaprasadv , we need it for security testing purpose of the API. We also want to use API test cases to run DAST scans using zap proxy.
